src/main/kotlin/no/nav/syfo/person/api/domain/syfomodiaperson/SyfomodiapersonBrukerinfo.kt

```diff
@@ -6,6 +6,8 @@ data class SyfomodiapersonBrukerinfo(
     val aktivPersonident: String,
     val navn: String? = null,
     val arbeidssituasjon: String = "ARBEIDSTAKER",
+    val kjonn: String?,
+    val fodselsdato: LocalDate?,
```

Comment:
Hva tenker vi om å feks levere alder her? Enten i tillegg eller i stedet for? Eller ønsker vi bare å levere data og heller regne ut i frontend?

Reply:
Kanskje lurt det (da kan vi også regne ut alder om `foedselsdato` mangler, men `foedselsaar` finnes.